client-code-gen/app-access-control-openapi.json

```diff
@@ -1 +1 @@
User Id"},"user_name":{"type":"string","maxLength":20,"title":"User Name"},"user_guid":{"anyOf":[{"type":"string","maxLength":32},{"type":"null"}],"title":"User Guid"},"create_user":{"type":"string","maxLength":60,"title":"Create User"},"update_user":{"anyOf":[{"type":"string","maxLength":60},{"type":"null"}],"title":"Update User"},"user_type":{"$ref":"#/components/schemas/FamUserType"}},"type":"object","required":["user_type_code","user_name","create_user","user_type"],"title":"FamUserOnlyName"},"FamUserRoleAssignmentCreate":{"properties":{"user_name":{"type":"string","maxLength":20,"minLength":3,"title":"User Name"},"user_type_code":{"$ref":"#/components/schemas/UserType"},"role_id":{"type":"integer","title":"Role Id"},"forest_client_number":{"anyOf":[{"type":"string","maxLength":8,"minLength":1},{"type":"null"}],"title":"Forest Client Number"}},"type":"object","required":["user_name","user_type_code","role_id"],"title":"FamUserRoleAssignmentCreate"},"FamUserRoleAssignmentGet":{"properties":{"user_role_xref_id":{"type":"integer","title":"User Role Xref Id"},"user_id":{"type":"integer","title":"User Id"},"role_id":{"type":"integer","title":"Role Id"},"application_id":{"type":"integer","title":"Application Id"}},"type":"object","required":["user_role_xref_id","user_id","role_id","application_id"],"title":"FamUserRoleAssignmentGet"},"FamUserType":{"properties":{"code":{"$ref":"#/components/schemas/UserType"},"description":{"type":"string","maxLength":35,"title":"Description"}},"type":"object","required":["code","description"],"title":"FamUserType"},"HTTPValidationError":{"properties":{"detail":{"items":{"$ref":"#/components/schemas/ValidationError"},"type":"array","title":"Detail"}},"type":"object","title":"HTTPValidationError"},"IdimProxyIdirInfo":{"properties":{"found":{"type":"boolean","title":"Found"},"userId":{"anyOf":[{"type":"string","maxLength":20},{"type":"null"}],"title":"Userid"},"displayName":{"anyOf":[{"type":"string","maxLength":50},{"type":"null"}],"title":"Displayname"}},"type":"object","required":["found"],"title":"IdimProxyIdirInfo"},"RoleType":{"type":"string","enum":["A","C"],"title":"RoleType"},"UserType":{"type":"string","enum":["I","B"],"title":"UserType"},"ValidationError":{"properties":{"loc":{"items":{"anyOf":[{"type":"string"},{"type":"integer"}]},"type":"array","title":"Location"},"msg":{"type":"string","title":"Message"},"type":{"type":"string","title":"Error Type"}},"type":"object","required":["loc","msg","type"],"title":"ValidationError"}},"securitySchemes":{"6jfveou69mgford233or30hmta":{"type":"oauth2","flows":{"authorizationCode":{"scopes":{},"authorizationUrl":"https://dev-fam-user-pool-domain.auth.ca-central-1.amazoncognito.com/authorize","tokenUrl":"https://dev-fam-user-pool-domain.auth.ca-central-1.amazoncognito.com/token"}}}}},"tags":[{"name":"Forest Access Management - FAM","description":"Controls the user access to different Forest basedapplications and what roles different users will have once logged in"}]}
```

Comment:
I don't see any change either on backend `router` or `schema` change in this PR. In normal case, it is not required generating new `openapi spec`.
But this time the spec has some difference.
Maybe it is due to the FastAPI version upgrade (internally the openapi it supports, since our swagger is created from FastAPI).
The change from below seems only on the javascript "Type" with optional "null". Which probably won't affect things in backend I feel. But on frontend, not sure if it would loose type checking. But should be fine.

Reply:
yeah, I feel there is no need to regenerate the client code, but then I notice the difference in OpenAPI spec, so I rerun it.